server/controllers/api/v1.js

```diff
@@ -44,7 +44,10 @@ class ApiRequestHandler {
   }
   async fetchFromWcl(cachedWclApiResponse) {
     const query = Object.assign({}, this.req.query, { api_key: this.apiKey });
-    const path = `v1/${this.req.params[0]}?${querystring.stringify(query)}`;
+    let path = `v1/${this.req.params[0]}?${querystring.stringify(query)}`;
+    if (path.substring(3, 19) === 'parses/character') { //re-encode URL for parses endpoint 
```

Comment:
MartijnHols commented 8 days ago:
```Actually if you're right that the issue is Express auto-decoding, your original solution would be closer to the best solution I can think of which would be disabling the auto-decoding of parameters for this endpoint. This would fix the underlying problem, while double-encoding is just a quick hack mostly```

Why I didn't fixed the auto-decoding issue is kinda explained in the PR

Reply:
ohh I meant to disable Express' auto-decoding, basically configuring the library's behavior to meet our needs.

Reply:
If you merge from master it should work without this piece of code and without the parts in the front-end that double encodes since the issue appears to be fixed.